src/account.ts

```diff
@@ -499,12 +499,15 @@ export class Account {
         { parse = parseJsonFromRawResponse } = {}
     ): Promise<any> {
         this.validateArgs(args);
+
+        const isUint8Array = args.byteLength !== undefined && args.byteLength === args.length;
```

Comment:
This way to detect if the type of `isUint8Array` was copied from other place in `near-api-js`. Some thoughts on this:

- We may want to put this function inside `utils` and reuse it elsewhere
- It is not very robust. For example, it detects Buffer (which "inherits" from Uint8Array), but it fails to detect properly `Uint8Array`. So we may as well call it `isBuffer`.

Reply:
Agree, let's not copy-paste it and move to a separate util.
I would not call args `serializedArgs` before they actually serialized.